docs/binder_governance.rst

```diff
@@ -0,0 +1,145 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communications, educational materials, and scientific work
```

Comment:
what do we mean by data science communications? why are we mentioning that first?

Reply:
For me it means "communication about science done with data" as well as what ever the reader thinks "data science" is.

I don't think the order implies a ranking, it just came out that way.